packages/base/src/3dview/mainview.tsx

```diff
@@ -1506,8 +1504,8 @@ export class MainView extends React.Component<IProps, IStates> {
           linewidth: DEFAULT_LINEWIDTH
         });
         const geometry = new THREE.BufferGeometry().setFromPoints([
-          explodedState.oldGeometryCenter,
-          explodedState.newGeometryCenter
+          groupMetadata.originalPosition.clone(),
+          group.position.clone()
```

Comment:
I think you should not change that. The line should go from the old geometry center to the new one, that seems to be the last remaining issue :)